android/src/toga_android/libs/utilfile.py

```diff
@@ -0,0 +1,58 @@
+import abc
+import io
+
+import java
+
+BufferedReader = java.jclass("java.io.BufferedReader")
+InputStreamReader = java.jclass("java.io.InputStreamReader")
+Objects = java.jclass("java.util.Objects")
+
+
+class HandlerFileDialog(abc.ABC):
```

Comment:
It's not clear to me why this is in `utilfile.py`, rather than part of the dialogs class.

Reply:
I was just afraid to change something in the main file.

Reply:
Ok... but it's a *dialog* class. It should be with *dialogs*, not with utilities for file handling.